tools/codegen/codespec/api_to_provider_spec_mapper_test.go

```diff
@@ -337,6 +337,12 @@ func TestConvertToProviderSpec_nested_schemaOverrides(t *testing.T) {
 									},
 								},
 							},
+							{
```

Comment:
is specifying a default timeout value out of scope at this point?

Reply:
Default timeout values are needed/configured in CRUD logic. Given that as of now they would not be used, would keep the model/config with the information necessary from schema generation aspect